cc3d/core/GraphicsUtils/CC3DPyGraphicsFrameIO.py

```diff
@@ -470,3 +470,30 @@ def control(self, control):
         """
         if self.blocking:
             self.control_self_blocking(control)
+
+
+class ControlMessageGetScreenshotData(FrameControlMessage):
+    """Message to get serialized screenshot data"""
+
+    def __init__(self):
+
+        super().__init__()
+
+        self.screenshot_data = None
+
+    def process(self, proc):
+        """Get screenshot data"""
+        self.screenshot_data = proc.frame.get_screenshot_data().to_json()
```

Comment:
Is there any chance of a `null` screenshot dict? I don't think so, but I want to check with you.